src/components/Modal/VotesList.tsx

```diff
@@ -0,0 +1,71 @@
+import React from 'react'
+import { navigate } from 'gatsby-plugin-intl'
+import { Modal, ModalProps } from 'decentraland-ui/dist/components/Modal/Modal'
+import Grid from "semantic-ui-react/dist/commonjs/collections/Grid/Grid"
+import Avatar from 'decentraland-gatsby/dist/components/User/Avatar'
+import { Address } from 'decentraland-ui/dist/components/Address/Address'
+import { Vote } from '../../entities/Votes/types'
+import { Header } from 'decentraland-ui/dist/components/Header/Header'
+import { Close } from 'decentraland-ui/dist/components/Close/Close'
+import TokenList from 'decentraland-gatsby/dist/utils/dom/TokenList'
+import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+import { abbreviateNumber } from '../../entities/Votes/utils'
+import locations from '../../modules/locations'
+
+import './ProposalModal.css'
+import './VotesList.css'
+
+export type VotesListModalProps = Omit<ModalProps, 'children'> & {
+  onClickAccept?: (e: React.MouseEvent<any>) => void
+  votes?: Record<string, Vote> | null,
+}
+
+export function VotesList({onClickAccept, votes, ...props }: VotesListModalProps) {
+  const l = useFormatMessage()
+
+  return <Modal {...props} size="tiny" className={TokenList.join(['ProposalModal', 'VotesList' ,props.className])} closeIcon={<Close />}>
+  <Modal.Content className="ProposalModal__Title">
+    <Header>{Object.entries(votes || {}).length} {Object.entries(votes || {}).length === 1 ? l('modal.votes_list.vote') : l('modal.votes_list.votes')}</Header>
```

Comment:
jajaj, I was specting to use a plural definition of intl:

```json5
    // en.json
    {
        "votes": "{votes} {votes, plural, one {Vote} other {Votes}}",
    }
```

and in code:
```jsx
    <Header>{l('modal.votes_list.votes', { votes: Object.keys(votes || {}).length })}</Header>
```

as ref:
https://github.com/decentraland/governance/blob/master/src/intl/en.json#L4